src/pages/study-room/components/participant-list.tsx

```diff
@@ -1,17 +1,54 @@
-import React from 'react';
+import { Button, List, Popover } from '@mui/material';
+import PeopleAltIcon from '@mui/icons-material/PeopleAlt';
+import { ParticipantSummary } from '@/types/study-room.types';
+import usePopover from '../hooks/usePopover';
 import ParticipantListItem from './participant-list-item';
-import { Card, CardContent, CardHeader } from '@mui/material';
 
-const ParticipantList = () => {
+interface ParticipantListProps {
+  participants: ParticipantSummary[];
+}
+
+const ParticipantList = ({ participants }: ParticipantListProps) => {
+  const { anchorEl, onOpen, onClose } = usePopover();
```

Comment:
커스텀훅에서 표현되는 함수의 행위 자체는 동사로 표현되는게 좋지 않을까요~?
```suggestion
  const { anchorEl, open, close } = usePopover();
```